js_build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine
```

Comment:
Don't use latest here.

If a new node version is published it will be a mismatch with the rest of the code.